wordpress.org/public_html/wp-content/plugins/plugin-directory/readme/class-parser.php

```diff
@@ -408,13 +416,18 @@ protected function parse_readme_contents( $contents ) {
 		// Use the first line of the description for the short description if not provided.
 		if ( ! $this->short_description && ! empty( $this->sections['description'] ) ) {
 			$this->short_description = array_filter( explode( "\n", $this->sections['description'] ) )[0];
+			$this->warnings['no_short_description_present'] = true;
```

Comment:
Since we pull fallback short description from main description, warning for `no_short_description_present` is not very useful I guess.

Reply:
That's true, I don't personally see it as something that should be flagged, but it's up to the client code as to whether to warn on it or not.

Reply:
Resolving this, as this is actually useful, see: https://github.com/WordPress/wordpress.org/pull/190/files#r1457051069